documentation/src/main/java/org/talend/runtime/documentation/Generator.java

```diff
@@ -185,7 +185,6 @@ public static void main(final String[] args) {
                 log.info("System is offline, skipping jira changelog and github contributor generation");
             } else {
                 tasks.register(() -> generatedContributors(generatedDir, args[5], args[6]));
-                tasks.register(() -> generatedJira(generatedDir, args[1], args[2], version));
```

Comment:
Hello,
Why do we change documentation generator in a version bump ticket please?

Reply:
We don't generate changelog from old jira anymore (don't have ticket in mind) and it may break build.